AI_Graphics/AI_Graphics/Preset.cs

```diff
@@ -51,14 +51,11 @@ public struct Preset {
                 try {
                     byte[] bytes = File.ReadAllBytes(targetPath);
                     Load(bytes);
-                    Debug.Log(string.Format("Loaded preset file '{0}'", name + ".preset"));
                     return true;
                 } catch {
-                    Debug.Log(string.Format("Failed to load preset file '{0}'", name + ".preset"));
                     return false;
                 }
             } else {
-                Debug.Log(string.Format("Couldn't find preset file '{0}'", name + ".preset"));
                 return false;
```

Comment:
I think we should keep debug log to see what's going on in case when save/load is just not working

Reply:
unless you know we have existing load/save issues, I just tested it, the debug is spamming the log file on the success case. I tested a number of failure cases and think we have covered all the basis. at a minimum, we should remove the first debug log when loading is fine, ok with that?

Reply:
Yes seems fine

Reply:
let me put back the debug logs for failure cases.